frontend/public/manifest.json

```
@@ -34,8 +36,14 @@
   "scope": "/",
   "start_url": "/pwa/start?utm_source=pwa",
   "display": "standalone",
+  "display_override": ["window-controls-overlay"],
+  "orientation": "natural",
   "theme_color": "#ffc800",
   "background_color": "#a09b87",
+  "launch_handler": {
+    "client_mode": ["focus-existing", "auto"]
```

it seems we prefer to keep the default
```suggestion
    "client_mode": ["auto"]
```